src/components/Debugger/DebuggerModal.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { Button, Modal, ModalVariant, TextContent } from '@patternfly/react-core';
+import { BugIcon } from '@patternfly/react-icons';
+import { TableComposable, Tbody, Th, Thead, Tr } from '@patternfly/react-table';
+import { ChromeUser } from '@redhat-cloud-services/types';
+import { useDispatch, useSelector } from 'react-redux';
+import { DeepRequired } from 'utility-types';
+
+import { toggleDebuggerModal } from '../../redux/actions';
+import { ReduxState } from '../../redux/store';
+import { Select, SelectList, SelectOption } from '@patternfly/react-core/next';
+import { MenuToggle, MenuToggleElement } from '@patternfly/react-core';
+
+import './Debugger.scss';
+
+export type DebuggerModalProps = {
+  user: DeepRequired<ChromeUser>;
+};
+
+const DebuggerModal = ({ user }: DebuggerModalProps) => {
+  const isOpen = useSelector<ReduxState, boolean | undefined>(({ chrome: { isDebuggerModalOpen } }) => isDebuggerModalOpen);
+  const dispatch = useDispatch();
+  const setIsModalOpen = (isOpen: boolean) => dispatch(toggleDebuggerModal(isOpen));
+  const handleCloseModal = () => {
```

Comment:
This function wrapper is not necessary. It's not doing anything extra. Setter can be called directly with the `false` argument.